arcgis-ios-sdk-samples/Maps/Display device location with NMEA data sources/README.md

```diff
@@ -6,9 +6,9 @@ Parse NMEA sentences and use the results to show device location on the map.
 
 ## Use case
 
-NMEA sentences can be retrieved from an MFi GNSS/GPS accessory and parsed into a series of coordinates with additional information.
+NMEA sentences can be retrieved from GPS receivers and parsed into a series of coordinates with additional information. Devices without a built-in GPS receiver can retrieve NMEA sentences by using a separate GPS dongle, commonly connected bluetooth or through a serial port.
```

Comment:
In https://github.com/Esri/arcgis-runtime-samples-ios/pull/1058#discussion_r616040205 (and other comments) Phil suggested these changes, but I didn't update the design README accordingly. So I think we may keep this README unchanged, while I can edit the design, if needed. 😳